### PR TITLE
Add more detailed rule description + rule examples

### PR DESCRIPTION
**Pull request by clemens-msupply**
Add some more rule details and provide some common examples.

### CODE REVIEW

**Review by sdirix — COMMENTED** [review]
Very nice! Thanks for the contribution :heart: 

Now that you added all these examples I would like to suggest to generify the previous initial example. Instead of using an example there we should show in an abstract way how a rule looks like, e.g.

```
"rule": {
  "effect": "HIDE" | "SHOW" | "ENABLE" | "DISABLE",
  "condition": {
    "scope": "<UI Schema scope>",
    "schema": JSON Schema
  }
}
```

Do you agree? Can you do this change too?

**Comment by clemens-msupply** [comment]
Good idea! I updated the branch

File: content/docs/uischema/rules.mdx [hunk 1]
```diff
@@ -10,22 +10,22 @@ import ApiLink from '../../../src/components/common/ApiLink';
...
 }
 ```
 
 A rule basically works by first evaluating the `condition` property and in case it evaluates to true, executing the associated `effect`.
 
-## RuleEffect
+
+## Rule Effect
 
 The `effect` property determines what should happen to the attached UI schema element once the `condition` is met. In the example above, if the `name` property has the value of `foo`, we'll hide the UI schema element the rule is attached to. Current effects include:
```
Comment (sdirix) [inline content/docs/uischema/rules.mdx]: This text also needs to be updated ;)

Comment (clemens-msupply) [reply content/docs/uischema/rules.mdx]: Ups missed that, thanks! I just removed the sentence think its clear enough without further description.

**Review by sdirix — APPROVED** [review]
Very nice improvement to the docs! Thanks for the contribution :heart:

**Pull request merged**